docs/docs/gatsby-cli.md

```diff
@@ -24,36 +23,38 @@ You can also use the `package.json` script variant of these commands, typically
 }
 ```
 
+## API commands
+
 ### `new`
 
-```shell
```

Comment:
![image](https://user-images.githubusercontent.com/50070488/66041797-6364f200-e538-11e9-9430-c14d7862c996.png)

removing `shell` will effect in these ? like it would be better to have a `shell` tag in those snippets

Reply:
'shell' or better 'bash' like the others

or something new: `syntax`/`defintion`? but `bash` would be the best